### DISCUSSION TITLE
Running Bounded Biharmonic Weights Tutorial

### DISCUSSION THREAD

**Discussion by mobereng** [libigl/libigl#1759] [General]
Hi guys, I'm trying to run Tutorial Example 403 (Bounded Biharmonic Weights) but on the tests files there is no `hand-pose.dmat` so my question is where can I find that file ?

Thanks in advance !

**Comment by jdumas** [comment]
It is in [libigl-tutorial-data](https://github.com/libigl/libigl-tutorial-data) which should be downloaded into `tutorial/data/` after you build the tutorials.

**Reply by mobereng** [reply]
Done ! thanks @jdumas
